app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -65,4 +85,54 @@ describe('Entry Component', () => {
       expect(screen.queryByTestId('global-spinner')).not.toBeInTheDocument();
     });
   });
+  test('should display "Make Pick" button when no pick is set', async () => {
```

Comment:
Can you share screenshots of the Make Pick buttons behavior?

Reply:
<img width="972" alt="Screenshot 2024-08-16 at 4 16 02 PM" src="https://github.com/user-attachments/assets/dafc15ee-f1fa-46e6-8d0d-2bb8d5fca054">